versions/1.0.0.md

```diff
@@ -92,12 +92,12 @@ This object represents one or more changes to be applied to the target document
 
 | Field Name | Type | Description |
 | ---- | :----: | ---- |
-| <a name="action-target"></a>target | `string` | **REQUIRED** A JSONPath query expression referencing the target objects in the target document. |
+| <a name="action-target"></a>target | `string` | **REQUIRED** A JSONPath expression selecting nodes in the target document. |
 | <a name="action-description"></a>description | `string` | A description of the action. [[CommonMark]] syntax MAY be used for rich text representation. |
-| <a name="action-update"></a>update | Any | An object with the properties and values to be merged with the object(s) referenced by the `target`. This property has no impact if `remove` property is `true`. |
+| <a name="action-update"></a>update | Any | If the `target` selects an object node, the value of this field should be an object with the properties and values to merge with the node. If the `target` selects an array, the value of this field should be an entry to append to the array. This field has no impact if the `remove` field of this action object is `true`. |
```

Comment:
Is the "merge" shallow or deep? That is, if `target` selects an object and `update` is an object with an object-valued property, is that property's object value _replaced_, or is it also (recursively) _merged_?

Reply:
On line 106 below it says:
> When the Overlay document is applied, the properties in the merge object replace properties in the target object with the same name and new properties are appended to the target object.

Based on that wording, I think the "merge" is actually a shallow merge.